### ISSUE TITLE
[LibreSSL] Build failure for deve/libevent

### ISSUE THREAD

**Issue by lattera** [HardenedBSD/hardenedbsd-ports#133]
Log file attached. Please consider this port high priority.

[2018-04-10-libevent-2.1.8_1.log.txt](https://github.com/HardenedBSD/hardenedbsd-ports/files/1894926/2018-04-10-libevent-2.1.8_1.log.txt)

**Comment by Sp1l** [comment]
Cannot reproduce... The patch that is in the HBSD repo applies fine for me...

**Comment by lattera** [comment]
On HardenedBSD 12-CURRENT/amd64:

```
hbsd-dev-laptop[shawn]:/usr/ports/devel/libevent $ make BATCH=1                                                                                                                    [12:49:03]
===>  License BSD3CLAUSE accepted by the user
===>   libevent-2.1.8_1 depends on file: /usr/local/sbin/pkg - found
=> libevent-libevent-release-2.1.8-stable_GH0.tar.gz doesn't seem to exist in /usr/ports/distfiles/.
=> Attempting to fetch https://codeload.github.com/libevent/libevent/tar.gz/release-2.1.8-stable?dummy=/libevent-libevent-release-2.1.8-stable_GH0.tar.gz
libevent-libevent-release-2.1.8-stable_GH0.tar100% of  684 kB  387 kBps 00m02s
===> Fetching all distfiles required by libevent-2.1.8_1 for building
===>  Extracting for libevent-2.1.8_1
=> SHA256 Checksum OK for libevent-libevent-release-2.1.8-stable_GH0.tar.gz.
===>  Patching for libevent-2.1.8_1
===>  Applying FreeBSD patches for libevent-2.1.8_1
1 out of 2 hunks failed--saving rejects to openssl-compat.h.rej
=> FreeBSD patch patch-libressl failed to apply cleanly.
=> Patch(es)  patch-gcc7 applied cleanly.
*** Error code 1

Stop.
make: stopped in /usr/ports/devel/libevent
hbsd-dev-laptop[shawn]:/usr/ports/devel/libevent (1) $ git status                                                                                                                  [12:49:09]
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
hbsd-dev-laptop[shawn]:/usr/ports/devel/libevent $ git log -n 1 | cat                                                                                                              [12:49:40]
commit 4b0a34da44e52c4deccf3326222c70fbd03916b4
Author: Bernard Spil <brnrd@FreeBSD.org>
Date:   Tue Apr 10 18:41:00 2018 +0200

    lang/python35: Fix build with LibreSSL 2.7
    
    Signed-off-by: Bernard Spil <bernard.spil@hardenedbsd.org>
```
